services/director-v2/src/simcore_service_director_v2/modules/db/repositories/groups_extra_properties.py

```diff
@@ -31,3 +38,15 @@ async def is_telemetry_enabled(self, *, user_id: int, product_name: str) -> bool
         )
         telemetry_enabled: bool = group_extra_properties.enable_telemetry
         return telemetry_enabled
+
+    async def get_user_extra_properties(
```

Comment:
Q: you do realise that you are breaking the pattern here, right?
I think it should remain as a single property, since it may not always be used as a group of 3.

Reply:
If you want to use this pattern, make sure you can delete the two above methods.

Reply:
I considered not optimal that 3 times in a row we would make the same query to the DB:
```
        group_extra_properties = await self._get_aggregated_properties_for_user(
            user_id=user_id, product_name=product_name
        )
```
which is done for each case of the thing you call a pattern. `GroupsExtraPropertiesRepository` is just an interface how to talk to groups extra properties table, I think it is OK to keep both there.